csrc/scheduler/matmul_utils.cpp

```diff
@@ -393,12 +394,34 @@ std::shared_ptr<MatmulParams> getMatmulHeuristics(
   // Disable magic zero for matmul kernels
   params->cparams.enable_magic_zero = false;
 
-  // Set whether to use shared memory for epilogue
   const auto& roles_map_opt =
       mma_utils::getTensorsRoles(fusion, mulSum.front().insouts);
   NVF_ERROR(roles_map_opt.isValid(), "Tensor roles map in mma is not valid.");
-
   const auto roles_map = roles_map_opt.getData();
+
+  const mma_utils::MatmulProblemLayoutOpt layout_opt =
+      mma_utils::getMmaLayout(fusion, mulSum.front().insouts);
```

Comment:
@protonu I added the second argument to `getMmaLayout`. It's not strictly related to this PR but I think it was wrong before and since I needed the layout for scheduling, I had to fix it. Can you confirm this is the correct pattern?

Reply:
That looks like the correct usage.
https://github.com/NVIDIA/Fuser/blob/52692ed5347dc308b93a7f35acdf2c4123c10c1f/csrc/scheduler/matmul_utils.cpp#L361-L368

Reply:
Right, that's what I was looking at. I just wanted to check that I was not modifying intentionally different behavior here.